src/sage/doctest/forker.py

```diff
@@ -946,15 +951,15 @@ def summarize(self, verbose=None):
         passed = []
         failed = []
         totalt = totalf = 0
-        for x in self._name2ft.items():
-            name, (f, t) = x
-            assert f <= t
-            totalt += t
-            totalf += f
-            if not t:
+        for x in self._stats.items():
+            name, f = x
```

Comment:
What's the advantage of changing `(f, t)` to `f` here?

Reply:
It is a triple in Python 3.13 so it can't be assigned to `(f,t)`

Reply:
This is another way to do it:
```suggestion
            name, (f, t, *_) = x
```
The `*_` will take zero or more entries so it's fine when `x[1]` is a pair and also when `x[1]` is a triple.

Reply:
Yes I feel the above is cleaner (or at least assign `f = u[0]; t = u[1]`). Need to change the parts below accordingly.